.env.sample

```diff
@@ -44,3 +43,12 @@ CLOUD_STORAGE_BUCKET_TYPE = public
 PRESIGNED_URL_EXPIRY_IN_SECONDS = 300                                           // presigned url expiry in seconds after which the signed url becomes invalid
 
 DOWNLOADABLE_URL_EXPIRY_IN_SECONDS = 300                                        // downloadable url expiry in seconds after which the downloadable url becomes invalid 
+
+
+#USER service
+USER_SERVICE_URL = "http://localhost:3001/user"    
+
+#EntityManagement service
+
+ENTITY_MANAGEMENT_SERVICE_URL = "http://localhost:3001/user"    
```

Comment:
@praveenKDass sample is confusing

Reply:
Changing to http://localhost:5001/entity @VISHNUDAS-tunerlabs